archinstall/default_profiles/desktops/sway.py

```diff
@@ -33,16 +33,13 @@ def packages(self) -> list[str]:
 		return [
 			"sway",
 			"swaybg",
-			"swaylock",
-			"swayidle",
-			"waybar",
-			"dmenu",
+			"wmenu",
 			"brightnessctl",
 			"grim",
-			"slurp",
-			"pavucontrol",
 			"foot",
-			"xorg-xwayland"
+			"xorg-xwayland",
+			"xdg-desktop-portal-gtk",
```

Comment:
is this essential? Seems to me more of an optional thing to work with things like Flatpak https://wiki.archlinux.org/title/XDG_Desktop_Portal